cmd/proxy/actions/app_proxy.go

```diff
@@ -47,7 +47,7 @@ func addProxyRoutes(
 		sumHandler := sumdbProxy(sumdbURL, c.NoSumPatterns)
 		pathPrefix := "/sumdb/" + sumdbURL.Host
 		r.PathPrefix(pathPrefix + "/").Handler(
-			http.StripPrefix(pathPrefix, sumHandler),
+			http.StripPrefix(strings.TrimRight(c.PathPrefix, "/")+pathPrefix, sumHandler),
```

Comment:
Out of curiosity, did you consider using `strings.TrimSuffix`? If a user has multiple slashes at the end of the PathPrefix, that is either a mistake they should realize themselves or maybe it's on purpose for whatever reason so we might not wanna magically remove all of them and replace them with the one "/". 

But happy to keep this as well 👍 

```suggestion
			http.StripPrefix(strings.TrimSuffix(c.PathPrefix, "/")+pathPrefix, sumHandler),
```